sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuShuffledHashJoinExec.scala

```diff
@@ -70,8 +70,24 @@ class GpuShuffledHashJoinMeta(
       (None, condition)
     }
     val Seq(left, right) = childPlans.map(_.convertIfNeeded())
+    val useSizedJoin = GpuShuffledSizedHashJoinExec.useSizedJoin(conf, join.joinType,
+      join.leftKeys, join.rightKeys)
     val joinExec = join.joinType match {
-      case Inner | FullOuter if conf.useShuffledSymmetricHashJoin =>
+      case LeftOuter | RightOuter if useSizedJoin =>
+        GpuShuffledAsymmetricHashJoinExec(
+          join.joinType,
+          leftKeys.map(_.convertToGpu()),
+          rightKeys.map(_.convertToGpu()),
+          joinCondition,
+          left,
+          right,
+          conf.isGPUShuffle,
+          conf.gpuTargetBatchSizeBytes,
+          isSkewJoin = false)(
+          join.leftKeys,
+          join.rightKeys,
+          conf.joinOuterMagnificationThreshold)
+      case Inner | FullOuter if useSizedJoin =>
```

Comment:
The config changed here. Is that intentional? (I assume that it is)

Reply:
Not sure what you mean by "config changed here."  The old code leveraged conf.useShuffledSymmetricHashJoin for inner and fullouter joins, and the new code is doing the same in  GpuShuffledSizedHashJoinExec.useSizedJoin.  Or did I miss something?